SharpTox/HL/Transfers/ToxTransfer.cs

```diff
@@ -12,15 +12,39 @@ public class ToxTransfer
         public string Name { get; private set; }
         public long Size { get; private set; }
         public ToxFileKind Kind { get; private set; }
-        public ToxTransferState State { get; protected set; } //initial state is 'paused' for the receiving end, 'in progress' for the sending end
+        public ToxTransferState State { get; protected set; }
+
+        private int _transferredBytes;
+
+        protected int TransferredBytes
```

Comment:
Perhaps we should make this property public but leave the setter protected. It would be nice to know how many bytes have been transferred.